### PR TITLE
Populate vsphere's kubeconfigs to secret/build-farm-credentials

### PR DESCRIPTION
**Pull request by hongkailiu**
The SAs are created by the postsubmit on vsphere.

Uploaded those files to BW already.

```
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.ci-operator.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.crier.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.deck.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.4K 14 Jul 15:25 sa.dptp-controller-manager.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.hook.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.plank.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.4K 14 Jul 15:25 sa.prow-controller-manager.vsphere.config
-rw-r--r--  1 hongkliu  staff   7.3K 14 Jul 15:25 sa.sinker.vsphere.config
```

/cc @openshift/openshift-team-developer-productivity-test-platform @jcpowermac

### CODE REVIEW

**Comment by hongkailiu** [comment]
@alvaroaleman 
This one is ready for review.
Thanks.

File: clusters/build-clusters/vsphere/ci-operator/admin_test-credentials.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```
Comment (alvaroaleman) [inline clusters/build-clusters/vsphere/ci-operator/admin_test-credentials.yaml:1]: Whats the purpose of all of this?

Comment (hongkailiu) [reply clusters/build-clusters/vsphere/ci-operator/admin_test-credentials.yaml:1]: Probably we wont run our ci-tools tests there on `vsphere`.

https://github.com/openshift/release/blob/ac2a12b7e8e33d360cc8ed9319277c0a9a7a2f86/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L349-L354

Comment (hongkailiu) [reply clusters/build-clusters/vsphere/ci-operator/admin_test-credentials.yaml]: Removed.

**Pull request merged**